process/sync/baseSync.go

```diff
@@ -64,16 +64,19 @@ type baseBootstrap struct {
 	syncStateListeners    []func(bool)
 	mutSyncStateListeners sync.RWMutex
 	uint64Converter       typeConverters.Uint64ByteSliceConverter
-	boostrapRoundIndex    uint32
+	bootstrapRoundIndex   uint32
 }
 
 func (boot *baseBootstrap) loadBlocks(
 	blockFinality uint64,
 	blockUnit dataRetriever.UnitType,
 	hdrNonceHashDataUnit dataRetriever.UnitType,
-	getHeader func(uint64) (data.HeaderHandler, []byte, error),
+	getHeader func(uint32, uint64) (data.HeaderHandler, []byte, error),
```

Comment:
can we have an interface here instead of multiple functions?

Reply:
Done